### ISSUE TITLE
Inputs in ActWidgets are not screen reader-friendly

### ISSUE THREAD

**Issue by baltpeter** [datenanfragen/website#480]
In the ActWidget, the form labels aren't set correctly. They refer to hidden inputs that don't actually do anything:

![image](https://user-images.githubusercontent.com/4048809/97886408-79874800-1d20-11eb-9f5a-b0da12e6f7cf.png)

This is because the ActWidget inherits from the generator where those hidden inputs are actually visible and allow the user to change the field description themselves.

**Comment by ForsakenHarmony** [comment]
I think you can get away with just the one real `input` and `label` without any extra `span` or `input`s

**Comment by baltpeter** [comment]
> I think you can get away with just the one real `input` and `label` without any extra `span` or `input`s

Yep, I agree. That's what we should have done in the first place. But this code is used for two (fairly different) cases where the other one does actually need the second input and we didn't consider that properly. Should now be fixed in #482.

> (You should also be able to just have one wrapping `div` instead of 3)

That's also a good point. I don't think we can really do anything about that right now as we are (unfortunately) still on Preact 8 without support for fragments but I'll open an another issue to fix that once we have finally upgraded.
